pkg/apis/serving/v1alpha1/revision_lifecycle.go

```diff
@@ -49,11 +49,11 @@ const (
 
 	// UserQueueMetricsPortName specifies the port name to use for metrics
 	// emitted by queue-proxy for end user.
-	UserQueueMetricsPortName = "user-metrics"
+	UserQueueMetricsPortName = "http-usrmetrics"
```

Comment:
```suggestion
	UserQueueMetricsPortName = "http-usermetrics"
```

Reply:
unfortunately, this will be picked-up then will bump to revision/reconcile_resources.go:48","msg":"Error creating deployment" invalid: spec.template.spec.containers[0].ports[0].name: Invalid value: "http-usermetrics": must be no more than 15 characters, this is by default by kubernetes. `http-usermetric` maybe.